core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala

```diff
@@ -18,21 +18,22 @@
 package kafka.admin
 
 import java.text.{ParseException, SimpleDateFormat}
+import java.util
 import java.util.{Date, Properties}
+
 import javax.xml.datatype.DatatypeFactory
 import joptsimple.{OptionParser, OptionSpec}
-
 import kafka.api.{OffsetFetchRequest, OffsetFetchResponse, OffsetRequest, PartitionOffsetRequestInfo}
 import kafka.client.ClientUtils
 import kafka.common.{OffsetMetadataAndError, TopicAndPartition}
 import kafka.consumer.SimpleConsumer
 import kafka.utils._
-import kafka.utils.Implicits._
-
 import org.I0Itec.zkclient.exception.ZkNoNodeException
 import org.apache.kafka.clients.CommonClientConfigs
+import org.apache.kafka.clients.admin.{AdminClient => NewAdminClient}
```

Comment:
@asasvari Other commands import o.a.k.clients.admin.AdminClient as JAdminClient. I believe being consistent in naming helps others understand and maintain this piece of code.